src/Microsoft.Azure.WebJobs.Host/Indexers/ExtensionTypeLocator.cs

```diff
@@ -10,6 +10,7 @@ internal class ExtensionTypeLocator : IExtensionTypeLocator
     {
         private readonly ITypeLocator _typeLocator;
         private IReadOnlyList<Type> _cloudBlobStreamBinderTypes;
+        private readonly static IReadOnlyList<Type> _emptyList = new List<Type>().AsReadOnly();
```

Comment:
Can't this entire ExtensionTypeLocator be removed? I think that'd be best, particularly since you've removed the link to the ICloudBlobStreamBinder type in this source, so we may forget to clean this up.

Reply:
This will already happen with the DI, and we wanted to minimize the merge conflicts.

Reply:
Indeed. As mentioned in the description, the proper fix (removal of the mentioned interfaces, etc.) is a bit invasive and will certainly introduce some painful conflicts. We'll address those once the DI changes are in place.